.gitlab-ci.yml

```diff
@@ -50,9 +50,8 @@ release:
     - branches
   script:
     - mkdir dist
-    - npm pack
```

Comment:
Will artifacts from previous job (build) be automatically available in this job?